examples/tide/main.rs

```diff
@@ -159,6 +159,11 @@ async fn main() -> tide::Result<()> {
             .with_cookie_name("webauthnrs");
     app.with(sessions);
     app.with(tide::log::LogMiddleware::new());
+    {
+        let prefix = prefix.clone();
+        app.at("/")
+            .get(move |_| async_std::future::ready(Ok(tide::Redirect::new(prefix.clone()))));
```

Comment:
You don't need this extra clone here since you have the prefix clone above.

Reply:
I believe without the extra clone we move ownership out of prefix which causes issues when we use it below.

Reply:
To make this clearer:

```
let prefix_copy = prefix.clone()
app.at().get(move |_| ... new(prefix_copy) 
```

This is clearer becuase it doesn't rely on shadowing the variable name, but also because the .get closure is move, then prefix copy will be moved there, meaning the second clone isn't needed, and the original prefix value is not affected.

Reply:
Oh sorry, I see you were talking about the clone within the closure. I believe that clone is necessary so the closure satisfies `Fn`, otherwise it only satisfies `FnOnce`.

Reply:
Do you mind pasting the compiler error?

Reply:
```
error[E0525]: expected a closure that implements the `Fn` trait, but this closure only implements `FnOnce`
   --> examples/tide/main.rs:164:14
    |
164 |         .get(move |_| async_std::future::ready(Ok(tide::Redirect::new(prefix_copy))));
    |          --- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^-----------^^^
    |          |   |                                                        |
    |          |   |                                                        closure is `FnOnce` because it moves the variable `prefix_copy` out of its environment
    |          |   this closure implements `FnOnce`, not `Fn`
    |          the requirement to implement `Fn` derives from here
```

Reply:
Well today I learnt. I'd still prefer you avoid the shadow naming, so would be good to rename let prefix to let prefix_copy or similar.

Reply:
pushed